com.milesalan.mepo.yml

```diff
@@ -0,0 +1,140 @@
+# Flatpak Manifest
+app-id: com.milesalan.mepo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ziglang
+extension-tag: '21.08'
+rename-icon: mepo
+rename-desktop-file: mepo.desktop
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-documents
+
+build-options:
+  - append-path: /usr/lib/sdk/ziglang
+command: mepo
+modules:
+  # Core build deps for mepo
+  - name: SDL2
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL/releases/download/release-2.24.0/SDL2-2.24.0.tar.gz
+        sha256: 91e4c34b1768f92d399b078e171448c6af18cafda743987ed2064a28954d6d97
+
+  - name: SDL2_image
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_image/releases/download/release-2.6.2/SDL2_image-2.6.2.tar.gz
+        sha256: 48355fb4d8d00bac639cd1c4f4a7661c4afef2c212af60b340e06b7059814777
+
+  - name: SDL2_ttf
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/libsdl-org/SDL_ttf/releases/download/release-2.20.1/SDL2_ttf-2.20.1.tar.gz
+        sha256: 78cdad51f3cc3ada6932b1bb6e914b33798ab970a1e817763f22ddbfd97d0c57
+
+  - name: SDL2_gfx
+    buildsystem: simple
+    build-commands:
+      - autoreconf -fvi
```

Comment:
this should build normailly with autotools. the autoreconf can go into a "shell" source

Reply:
Fixed